include/quicly/loss.h

```diff
@@ -215,8 +222,6 @@ inline void quicly_loss_update_alarm(quicly_loss_t *r, int64_t now, int64_t last
     if (r->loss_time != INT64_MAX) {
         /* time-threshold loss detection */
         alarm_duration = r->loss_time - last_retransmittable_sent_at;
-    } else if (r->rtt.smoothed == 0) {
-        alarm_duration = 2 * r->rtt.latest; /* should contain initial rtt */
```

Comment:
We no longer need this special-case, because we can consistently rely on `quicly_rtt_get_pto`.

This change also fixes the bug in the existing code that failed to apply exponential increase before obtaining the first RTT sample.